pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -4556,13 +4644,75 @@ SKIP :
 	" " | "\t" | "\f"
 }
 
+/* 
+This Grammar (or JavaCC) has several bugs.
+
+Currently (2021-02-26) it fails for things like
+- dbms_lob.trim(...),
+- scalar subqueries in the WHERE-clause,
+- TREAT(object AS type),
+just to name a few.
+
+Running PMD over a code base of approx. 1100 DDL scripts
+from a commercial product, the parser failed for 182 files.
+About 10% of these parsing errors actually showed flaws in the code,
+e.g. "\u2013" instead of "-", or utf-8 encoding instead of windows-1252
+(which is the standard for the product's scripts).
+Still, ~ 15% of perfectly valid DDL scripts could not be parsed by PMD.
+
+Nevertheless it should be best practice to call PMD for _every_ DDL script.
+
+Thus, we introduce the following workaround to cope with the situation.
```

Comment:
These explanations are very valuable. I guess, we should add these to the language specific documentation for PLSQL on https://pmd.github.io/latest/ (there is currently no specific page for plsql).

You can create a new markdown page here: https://github.com/pmd/pmd/tree/master/docs/pages/pmd/languages
And add this new page in the sidebar here: https://github.com/pmd/pmd/blob/342dc1d03aaa1082e42f7496d6869d15859af321/docs/_data/sidebars/pmd_sidebar.yml#L340